chapters/dump-reload.rst

```diff
@@ -0,0 +1,127 @@
+Rolling Updates and Query While Indexing in Jina
+================================================
+
+In this section, you will learn how to configure Jina to allow queries while indexing
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Overview
+^^^^^^^^
+
+Querying while indexing refers to the capacity of still being able to
+search your system while data is simultaneously being written to it.
+Within Jina this is possible using the rolling updates feature. This
+means you can have two Flows running at the same time, one for DBMS-type
+operations (adding, updating, deleting data) and one for searching
+(querying). The two systems communicate with each other using a system
+of rolling update calls to the respective Flows.
+
+======================= ==========================================
+**Section**             **Purpose**
+======================= ==========================================
+DBMS and Query Indexers Understand the two Flows you will need
+Dump-Reload             The methods for interacting with the Flows
+Replicas                Configuration needed for uptime
+JinaD                   Remote Flows
+======================= ==========================================
+
+DBMS and Query Indexers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/dump-reload/dbms-query.jpg
```

Comment:
these images are huge.  You should be able to adjust them to something more reasonable? 

<img width="1061" alt="Screenshot 2021-05-17 at 13 41 02" src="https://user-images.githubusercontent.com/59612379/118482823-8ea05300-b715-11eb-9ece-c97bc98e80c6.png">